src/simplewallet/simplewallet.cpp

```diff
@@ -5656,6 +5723,67 @@ bool simple_wallet::prompt_if_old(const std::vector<tools::wallet2::pending_tx>
   }
   return true;
 }
+void simple_wallet::check_for_inactivity_lock(bool user)
+{
+  if (m_locked)
+  {
+#ifdef HAVE_READLINE
+    PAUSE_READLINE();
+    rdln::clear_screen();
+#endif
```

Comment:
```suggestion
#else
    tools::clear_screen();
#endif
```

Reply:
Why ? That makes it less likely to work, no ?

Reply:
TBH i'm fine with both ways. Feel free to leave it as is if you want.